modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/twostep/GridMapQueryExecutor.java

```diff
@@ -582,15 +633,18 @@ private void onQueryRequest0(
                         if (qryRetryErr != null)
                             sendError(node, reqId, qryRetryErr);
                         else {
+                            String errMsg = buildQueryLogDetails(reqId, label, schemaName, qrys, params, e, node.id(),ctx.localNodeId());
+
                             if (e instanceof Error) {
-                                U.error(log, "Failed to execute local query.", e);
+                                U.error(log, errMsg, e);
 
                                 throw (Error)e;
                             }
+                            else {
```

Comment:
There is no point in adding "else" here. Also, let's change the logging level back to WARN here, since it was intentionally downgraded sometime ago as part of GG-34383.